lab/software-stack: Fix typos and grammar errors (#163)
open-education-hub/operating-systems: content/chapters/software-stack/lab/content/high-level-lang.md

```

We can see that on all metrics, the running of the Python program is less efficient than the running of the C program.

> `69` milliseconds are used by Python, whereas less than `1` millisecond for the C program.

Reviewer: ```suggestion
The Python code takes `69` milliseconds, whereas the C code runs in less than `1` millisecond.
```
